experimenter/experimenter/changelog/views.py

```diff
@@ -1 +1,27 @@
-# Create your views here.
+from django.shortcuts import get_object_or_404
+from django.views.generic import DetailView
+
+from experimenter.experiments.models import NimbusExperiment
+
+
+class NimbusChangeLogsView(DetailView):
+    model = NimbusExperiment
+    template_name = "changelog/overview.html"
+    context_object_name = "experiment"
```

Comment:
This should be unnecessary:

> Well, if you’re dealing with a model object, this is already done for you. When you are dealing with an object or queryset, Django is able to populate the context using the lowercased version of the model class’ name.

[ref](https://docs.djangoproject.com/en/4.2/topics/class-based-views/generic-display/#making-friendly-template-contexts)